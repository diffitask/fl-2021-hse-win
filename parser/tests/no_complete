Alphabet: 2 -- ("a", "b")
Q: 2
start: 1
runoff: 0
T: 1 -- 2
edges: (1; 2; "a", "b")