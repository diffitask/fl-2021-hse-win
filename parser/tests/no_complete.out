Error: Automat is not complete -- no edge with symbol 'b' from state 2
